core/xchain/client_simulator/client_simulator.go

```diff
@@ -17,110 +16,15 @@ import (
 	node_crypto "github.com/river-build/river/core/node/crypto"
 	"github.com/river-build/river/core/node/dlog"
 
-	xc "github.com/river-build/river/core/xchain/common"
-
 	"github.com/river-build/river/core/contracts/base"
 	"github.com/river-build/river/core/contracts/base/deploy"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/ethclient"
 )
 
-var isEntitled = false
```

Comment:
I plan to remove most of what's in this file (see https://github.com/river-build/river/issues/1081) but for now, to keep the PR appropriately scoped, I am just removing ISENTITLED functionality.